build.gradle

```diff
@@ -20,13 +20,18 @@ buildscript {
     repositories {
         mavenCentral()
     }
+    dependencies {
```

Comment:
This is not how we are configuring plugins. Kindly follow the conventions. 

- Plugin configuration goes in `pluginManagement { ` 
- plugin version goes in gradle.properties file. 
- We do not use apply, kindly see above.

Reply:
removed up